wurst/systems/modes/Respawn.wurst

```diff
@@ -0,0 +1,61 @@
+package Respawn
+import RegisterEvents
+import TrollExtensions
+import TimerExtensions
+import ClosureTimers
+import GameConfig
+import TimerUtils
+import TimerDialog
+
+@configurable constant GRACE_PERIOD_REVIVE_DELAY = 10.0
+
+function isGracePeriodActive() returns boolean
+    return udg_NOOB_TIME
+
+function quadraticReviveDelay(real x) returns real
```

Comment:
Change `x` to `seconds` or `num_seconds`. I'd prefer the latter but it's whatever.

Reply:
We use `camelCase` not `snake_case` so I'll switch it to `numSeconds` and `numMinutes`